src/renderers/shaders/ShaderChunk/bsdfs.glsl

```diff
@@ -62,7 +62,9 @@ float G_GGX_Smith( const in float alpha, const in float dotNL, const in float do
 
 	float gv = dotNV + pow( a2 + ( 1.0 - a2 ) * pow2( dotNV ), 0.5 );
 
-	return 1.0 / ( gl * gv );
+	float denominator = ( gl * gv ) + 0.00001; // avoid possible divide by 0 on the next line
```

Comment:
I have only seen this issue arise on low precision mobile devices.  a2 = pow4(0.04) = 0.00000256.  Mediump float range is [2^-14, 2^14] according to the spec.  Many mobile devices are limited to mediump float precision.   That means the smallest number is 0.00006103515, which is larger than pow4(0.04), which means pow4(0.04) will round to zero.  I believe this explains the issue.